rpcs3/CMakeLists.txt

```diff
@@ -181,9 +181,9 @@ set(CMAKE_MODULE_PATH "${RPCS3_SRC_DIR}/cmake_modules")
 find_package(OpenGL REQUIRED)
 find_package(OpenAL REQUIRED)
 if(NOT WITHOUT_LLVM)
-	find_package(LLVM 4.0 CONFIG)
+	find_package(LLVM 999.666 CONFIG)
```

Comment:
I guess the intention here was to cause this check to always fail since any system LLVM is not longer suited for RPCS3, is that correct? Wouldn't be better just to remove the `if` at line 185?

Reply:
If it's removed it'll be harder to put it back in the case someone wants to try installed package.